gui/seedwatch.lua

```diff
@@ -0,0 +1,310 @@
+-- config ui for seedwatch
+
+local gui = require('gui')
+local widgets = require('gui.widgets')
+local plugin = require('plugins.seedwatch')
+
+local PROPERTIES_HEADER = '   Quantity     Target   '
+local REFRESH_MS = 10000
+local MAX_TARGET = 2147483647
+--
+-- SeedSettings
+--
+SeedSettings = defclass(SeedSettings, widgets.Window)
+SeedSettings.ATTRS{
+    lockable=false,
```

Comment:
This attribute only existed for a brief time and no longer exists. I just removed other leftover references to it (gui/autochop and gui/automelt)

Reply:
Removed lockable